cmd/sriov/main.go

```diff
@@ -43,6 +44,18 @@ func getEnvArgs(envArgsString string) (*envArgs, error) {
 }
 
 func cmdAdd(args *skel.CmdArgs) error {
+	// Set logging related info. The default log level is error, hence also do logging.Debug inside the block.
+	if logLevel, logFile, err := config.LoadLoggingConf(args.StdinData); err == nil {
```

Comment:
I think it's better to create a function to setup the logging. It could be created in config.go. Of course, the debug log should be the only one that remains in cmdAdd and cmdDel.

Reply:
perhaps, you should only set the logLevel and logFile when these are not ""

Reply:
I prefer functions be omnipotent though and not rely too much on the defaults of the underlying library

Reply:
fair